### PR TITLE
FEV-1660 Show the "multiple pics" alert only if previewFramesSampleRate > 0

### PR DESCRIPTION
**Pull request by tpaik18**
Also, follow-on fix for FEV-1659. It wasn't working properly because I forgot to `await`.

### CODE REVIEW

File: FluStudy_us/src/ui/components/flu/AndroidRDTReader.tsx [hunk 1]
```diff
@@ -145,9 +145,11 @@ interface State {
 }
 
 class AndroidRDTReader extends React.Component<Props & WithNamespaces, State> {
+  _sampleRate: number = getRemoteConfig("previewFramesSampleRate");
```
Comment (audererob) [inline FluStudy_us/src/ui/components/flu/AndroidRDTReader.tsx]: Can be const

File: FluStudy_us/src/ui/components/flu/AndroidRDTReader.tsx [hunk 2]
```diff
@@ -145,9 +145,11 @@ interface State {
...
+
   state: State = {
-    spinner: false,
-    showCamera: false,
+    spinner: this._sampleRate > 0 ? false : true,
```
Comment (audererob) [inline FluStudy_us/src/ui/components/flu/AndroidRDTReader.tsx]: Nit: could be spinner: this._sampleRate === 0,
(arguable if that's more/less readable)

File: FluStudy_us/src/ui/components/flu/AndroidRDTReader.tsx (hunk 1 above)
Comment (tpaik18) [reply FluStudy_us/src/ui/components/flu/AndroidRDTReader.tsx]: For some reason when I add `const` to the front Prettier strips it away.

Comment (rightparen) [reply FluStudy_us/src/ui/components/flu/AndroidRDTReader.tsx]: @tpaik18 fields use `readonly`, locals use `const`.

**Pull request merged**